src/index.html

```diff
@@ -3,12 +3,38 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Roboto:wght@500&display=swap">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo-link" href="#"> <img class="logo" src="images/logo.png"
+         alt="LOGO">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list__item"> <a class="nav__list__link is-active" href="#"> apple </a> </li>
+          <li class="nav__list__item"> <a class="nav__list__link" href="#"> samsung </a> </li>
+          <li class="nav__list__item"> <a class="nav__list__link" href="#"> smartphones </a> </li>
+          <li class="nav__list__item"> <a class="nav__list__link" href="#"
+             data-qa="hover"> laptops & computers </a>
+          </li>
+          <li class="nav__list__item"> <a class="nav__list__link" href="#"> gadgets </a> </li>
+          <li class="nav__list__item"> <a class="nav__list__link" href="#"> tablets </a> </li>
+          <li class="nav__list__item"> <a class="nav__list__link" href="#"> photo </a> </li>
+          <li class="nav__list__item"> <a class="nav__list__link" href="#"> video </a> </li>
```

Comment:
You don't need to use 2 tags (li and a) for these items. Leave a, because it's a link, delete li. Move class and/or css-properties, if you need

Reply:
We did it with 'li' on a courses and I think it's not a problem. It easer to watch like a structure 'ul'.